docs/integrations/app-development/opentelemetry/jfrog-artifactory-opentelemetry.md

```diff
@@ -0,0 +1,175 @@
+```
+ </TabItem>
+<TabItem value="Windows">
+
+1.  Copy the yaml file to `C:\ProgramData\Sumo Logic\OpenTelemetry Collector\config\conf.d` folder in the machine which needs to be monitored.
+2.  Restart the collector using
+```sh
+Restart-Service -Name OtelcolSumo
+```
+</TabItem>
+<TabItem value="macOS">
+
+1.  Copy the yaml file to `/etc/otelcol-sumo/conf.d/` folder in the Artifactory instance which needs to be monitored.
+2.  Restart the otelcol-sumo process using the below command
+```sh
+otelcol-sumo --config /etc/otelcol-sumo/sumologic.yaml --config "glob:/etc/otelcol-sumo/conf.d/*.yaml"
+```
+
+</TabItem>
+</Tabs>
+
+After successful execution of the above command, Sumo will start receiving the data from your host machine.
+
+This will install the app to your Sumo Logic Org. The app consists of Dashboards.
+
+Panels will start to fill automatically. It's important to note that each panel slowly fills with data matching the time range query and received since the panel was created. Results won't immediately be available, but within 20 minutes, you'll see full graphs and map
+
+### Sample Log Messages in Non-Kubernetes environments
+
+```
+2023-45-16 11:45:44,171 [a8bgdia2di2g80kh] [ACCEPTED DEPLOY] hortonworks-cache:org/apache/hadoop/hadoop-project/2.6.0-cdh5.4.4-SNAPSHOT/maven-metadata.xml for client : admin/195.186.216.125.
+```
+
+### Sample Query
+
+This sample Query is from the **Artifactory - Artifactory - Cached Deployment Activity > Accepted Deploys** by Geolocation panel.
+
+Query String
+
+```sql
+" %"sumo.datasource"=artifactory "ACCEPTED DEPLOY" "-cache"
+|parse "[*] [*] *" as trace_id, event_type, user_info
+| parse regex field=user_info "(?:(?<repo>[^:]*):(?<path>[^\s]*))?\s+(?<opt_msg>[\w\s:]+)?\s+(?<user>[^\/]+)\/(?<ip>\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})\."
+| where event_type = "ACCEPTED DEPLOY" and repo matches "*-cache"
+| count  by ip | sort _count
+| lookup longitude, latitude from geo://location on ip = ip
+"
+```
+
+## Viewing IIS Dashboards
+
+### Artifactory-Overview
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Artifactory-OpenTelemetry/Artifactory-Overview.png')} alt="Artifactory-Overview" />
+
+### Artifactory - Cached Deployment Activity
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Artifactory-OpenTelemetry/Artifactory-Cached-Deployment-Activity.png')} alt="Artifactory-Cached-Deployment-Activity" />
+
+### Artifactory - Download Activity
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Artifactory-OpenTelemetry/Artifactory-Download-Activity.png')} alt="Artifactory-Download-Activity" />
+
+### Artifactory - Non-Cached Deployment Activity
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Artifactory-OpenTelemetry/Artifactory-Non-Cached-Deployment-Activity.png')} alt="Artifactory-Non-Cached-Deployment-Activity" />
+
+### Artifactory - Request and Access
```

Comment:
Content under this heading needs to be copied from - https://help.sumologic.com/docs/integrations/app-development/jfrog-artifactory/#request-and-access